sdk/anomalydetector/azure-ai-anomalydetector/src/test/resources/session-records/testDetect.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
This file did not make it to the assets repo, intentional?

Reply:
This was an old version of recording that was no longer used but never deleted.